### PR TITLE
Add zig

### PR DESCRIPTION
**Pull request by yhyadev**
Simply, I have added zig file that outputs "Hello, world!" in the terminal.

### CODE REVIEW

**Comment by yhyadev** [comment]
> You don't need to have `,` in the hello world sentence just like other languages.

I have seen that in the assembly one

File: hello-world.zig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const std = @import("std");
+
+pub fn main() !void {
+    const stdout = std.io.getStdOut().writer();
+
+    try stdout.print("Hello, world!\n", .{});
```
Comment (AbdelrahmanDwedar) [inline hello-world.zig]:
```zig

**Comment by yhyadev** [comment]
But ok, i will change it

**Review by AbdelrahmanDwedar — APPROVED** [review]
LGTM

**Pull request merged**